proxy/src/telemetry/control.rs

```diff
@@ -24,6 +24,8 @@ pub struct MakeControl {
     flush_interval: Duration,
 
     process_ctx: Arc<ctx::Process>,
+
+    prometheus_labels: Option<Arc<str>>,
```

Comment:
do you think it makes sense to just store this in the process ctx? nearly everywhere we reference it, we also have a `process_ctx` already.